src/schema/draft/manifestation.js

```diff
@@ -824,10 +824,16 @@ export const resolvers = {
           materialType: parent?.materialTypes?.[0]?.specific,
           colors,
         };
-        coverImage = await context.datasources
+        const defaultForsiderCoverImage = await context.datasources
           .getLoader("defaultForsider")
           .load(params);
-        return coverImage || {};
+
+        if (
+          defaultForsiderCoverImage &&
+          Object.keys(defaultForsiderCoverImage).length > 0
```

Comment:
er det noget specifikt du har brug for som f. ex. defaultForsiderCoverImage.url? så vil en defaultForsiderCoverImage?.url være mere explicit